docs-next/src/components/ClientRedirects.astro

```diff
@@ -0,0 +1,198 @@
+    "no-browser-support": "features/parallel-mode/#no-browser-support",
+    "nodejs-native-esm-support": "explainers/nodejs-native-esm-support",
+    nyan: "reporters/nyan",
+    "options-that-differ-slightly-from-cli-options":
+      "running/browsers/#options-that-differ-slightly-from-cli-options",
+    "options-that-only-function-in-browser-context":
+      "running/browsers/#options-that-only-function-in-browser-context",
+    "parallel-mode": "explainers/run-cycle-overview#parallel-mode",
+    "parallel-mode-worker-ids":
+      "features/parallel-mode/#parallel-mode-worker-ids",
+    "parallel-tests": "features/parallel-mode",
+    "pending-tests": "declaring/pending-tests",
+    priorities: "running/configuring/#priorities",
+    progress: "reporters/progress",
+    qunit: "interfaces/qunit",
+    "reporter-limitations": "features/parallel-mode/#reporter-limitations",
+    reporters: "reporters/about",
+    reporting: "running/browsers/#reporting",
+    require: "interfaces/require",
+    "retry-tests": "declaring/retrying-tests",
+    "root-hook-plugins": "features/root-hook-plugins",
+    "root-hook-plugins-can-export-a-function":
+      "features/root-hook-plugins#root-hook-plugins-can-export-a-function",
+    "root-hooks-are-not-global":
+      "features/parallel-mode/#root-hooks-are-not-global",
+    "root-level-hooks": "features/hooks/#root-level-hooks",
+    "run-cycle-overview": "explainers/run-cycle-overview",
+    "running-mocha-in-the-browser": "running/browsers",
+    "serial-mode": "explainers/run-cycle-overview#serial-mode",
+    spec: "reporters/spec",
+    sponsors: "#sponsors",
+    "suite-level": "features/timeouts#suite-level",
+    "synchronous-code": "features/asynchronous-code/#synchronous-code",
+    tap: "reporters/tap",
+    tdd: "interfaces/tdd",
+    "test-duration": "explainers/test-duration",
+    "test-duration-variability":
+      "features/parallel-mode/#test-duration-variability",
+    "test-fixture-decision-tree-wizard-thing":
+      "explainers/test-fixture-decision-tree",
+    "test-level": "features/timeouts#test-level",
+    textmate: "running/editor-plugins/#textmate",
+    "the-test-directory": "running/test-globs",
+    "third-party-reporters": "reporters/third-party",
+    timeouts: "features/timeouts",
+    "troubleshooting-parallel-mode":
+      "features/parallel-mode/#no-browser-support",
+    "using-async-await": "features/asynchronous-code/#using-async--await",
+    wallabyjs: "running/editor-plugins/#wallabyjs",
+    "when-not-to-use-global-fixtures":
+      "features/global-fixtures/#when-not-to-use-global-fixtures",
+    "when-to-use-global-fixtures":
+      "features/global-fixtures/#when-to-use-global-fixtures",
+    "with-commonjs": "features/root-hook-plugins#with-commonjs",
+    "with-es-modules": "features/root-hook-plugins#with-es-modules",
+    "working-with-promises":
+      "features/asynchronous-code/#working-with-promises",
+    xunit: "reporters/xunit",
```

Comment:
For the record, devtools elements search of `[id` finds 224 matches on mochajs.org compared to the ~150 here, but this is definitely a good start! Heck, maybe my search is overzealous, we can discuss further in the issue itself

![image](https://github.com/user-attachments/assets/af6ac491-9530-4f9b-baa0-57f15a4e2192)

Reply:
Back-linking to the explanation: https://github.com/mochajs/mocha/issues/5306#issuecomment-2791352025 👍